website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,34 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## Configuring the JWT auth method to avoid API calls to the Kubernetes API Server
+
+Vault Secrets Operator supports using the [JWT auth method](/vault/docs/platform/k8s/vso/api-reference#vaultauthconfigjwt).
+In the default JWT configuration this involves an API call from the Vault server back to the Kubernetes API server to [verify](/vault/docs/auth/jwt#jwt-verification) the JWT.
+It is possible to configure Vault to use [`jwt_validation_pubkeys`](/vault/api-docs/auth/jwt#jwt_validation_pubkeys) fields of the JWT auth configuration
+to work in the Kubernetes environment and avoid the API call back to Kubernetes, improving performance of login requests.
+
+### Configure the JWT Auth method in Vault to use the JWKS keys from Kubernetes:
+
+* Fetch the JWKS key from Kubernetes using `kubectl`
+
+```shell
```

Comment:
I think we typically use `shell-session` for command line examples
```suggestion
```shell-session
```